notebooks/Misconfiguration_detection_framework_for_data_type_errors.ipynb

```diff
@@ -0,0 +1,821 @@
+{
```

Comment:
<p>Can you move all of the imports to the same cell above?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/configuration-files-analysis/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
Hi @MichaelClifford , 
I moved all of the imports to the top of the same cell.

Reply:
<p>&nbsp;&nbsp;if not scheme or scheme:</p><p>&nbsp;&nbsp;&nbsp;&nbsp;return False</p><p> </p><p>Is this code correct?</p>

Reply:
my bad. I made some mistake in final version.
it should be `if not scheme or not domain:`

Reply:
I fixed this issue